app/code/Magento/CmsGraphQl/etc/schema.graphqls

```diff
@@ -32,10 +32,15 @@ type CmsPage @doc(description: "CMS page defines all CMS page information") {
 
 type CmsBlocks @doc(description: "CMS blocks information") {
     items: [CmsBlock] @doc(description: "An array of CMS blocks")
+    errors: [ErrorMessage] @doc(description: "An array of errors message")
```

Comment:
We do not need to introduce "errors" for specific queries. "errors" functionality is available out of the box. All you need to do is to throw `\Magento\Framework\GraphQl\Exception\GraphQlInputException` or another exception which implements `\GraphQL\Error\ClientAware` from your resolver.